app/code/community/Metrilo/Analytics/Api/Connection.php

```diff
@@ -0,0 +1,68 @@
+<?php
+class Metrilo_Analytics_Api_Connection
+{
+    /**
+     * Create HTTP POST request to URL
+     *
+     * @param String $url
+     * @param Array $bodyArray
+     * @param Boolean $hmacAuth
+     * @return void
+     */
+    public function post($url, $bodyArray = false, $hmacAuth = false)
+    {
+        $parsedUrl = parse_url($url);
+        $headers = [
+            'Content-Type: application/json',
```

Comment:
`Content-Type` for the new tracking API is `text/plain`. Ref: https://github.com/Metrilo/custom-integration